src/YoutubeDl.php

```diff
@@ -139,7 +139,10 @@ private function jsonDecode($data): array
 
     private function processDownload(Process $process): Video
     {
-        $videoData = $this->jsonDecode(trim($process->getOutput()));
+        $jsonOutput = str_replace(['"','%(ext)s'],['','info.json'],$this->options['output']);
+        $jsonOutput = file_get_contents($this->getDownloadPath().'/'.$jsonOutput);
```

Comment:
In master branch there is no `getDownloadPath` method, please use property directly